gensim/models/keyedvectors.py

```diff
@@ -1036,33 +835,11 @@ def init_sims(self, replace=False):
         effectively read-only = you can call `most_similar`, `similarity` etc., but not `train`.
 
         """
-        if getattr(self, 'syn0norm', None) is None or replace:
+        if getattr(self, 'vectors_norm', None) is None or replace:
             logger.info("precomputing L2-norms of word weight vectors")
             if replace:
-                for i in xrange(self.syn0.shape[0]):
-                    self.syn0[i, :] /= sqrt((self.syn0[i, :] ** 2).sum(-1))
-                self.syn0norm = self.syn0
+                for i in xrange(self.vectors.shape[0]):
+                    self.vectors[i, :] /= sqrt((self.vectors[i, :] ** 2).sum(-1))
+                self.vectors_norm = self.vectors
             else:
-                self.syn0norm = (self.syn0 / sqrt((self.syn0 ** 2).sum(-1))[..., newaxis]).astype(REAL)
-
-    def get_keras_embedding(self, train_embeddings=False):
```

Comment:
Where this now?

Reply:
This is in `~gensim.models.word2vec.Word2VecKeyedVectors` now. Should I shift this to `~gensim.models.keyedvectors.WordEmbeddingsKeyedVectors`. This shift would only be required if `FastText` models want to use this method. @menshikh-iv